resources/views/components/game-info.blade.php

```diff
@@ -16,6 +16,19 @@
         <p>You already have {{$game->title}} on your Wishlist PEASANT, wouldst thou like to remove?</p>
     </div>
 
+@auth
+
+    <h2>Leave a Review!</h2>
+    <form>
```

Comment:
we're going to need a few attributes on the form tag like method and action. i suppose action can wait until you've created a route for the form submit. we also need `@csrf` for the csrf token

Reply:
to be clear, method should be POST

Reply:
copy that!